java-examples/src/main/java/edu/practice/algos/sort/impl/MergeSort.java

```diff
@@ -0,0 +1,59 @@
+package edu.practice.algos.sort.impl;
+
```

Comment:
add javadocs on what this class does like:
/**
* This class sorts an array.
*
*/

Add this to all classes.

Reply:
kk